### PR TITLE
commit classes and add pom.xml

### PR DESCRIPTION
**Pull request by ViktorMarchuk**

### CODE REVIEW

File: A.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 public class A {
     public static void main(String[] args) {
         System.out.println("Hello! How are your?");
+        System.out.println("First change");
```
Comment (dmitriystel) [inline A.java:4]: add in string 5 - System.out.println(done");

Comment (ViktorMarchuk) [reply A.java:4]: ok, leave it

Comment (dmitriystel) [inline A.java:4]: delete string nr 3

File: B.java [hunk 2]
```diff
@@ -23,15 +23,6 @@ public boolean equals(Object o) {
...
-    }
-
-    @Override
-    public String toString() {
-        return "B{" +
-                "id=" + id +
-                '}';
-    }
 }
+
```
Comment (dmitriystel) [inline B.java:27]: add field address in the class

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <groupId>org.example</groupId>
+    <artifactId>demo-pr</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+    </properties>
+
+</project>
```
Comment (dmitriystel) [inline pom.xml:16]: well done, it might be like this!

File: A.java (hunk 1 above)
Comment (dmitriystel) [reply A.java:4]: remove strint nr 2

**Pull request merged**